lib/kino/proxy.ex

```diff
@@ -46,8 +44,28 @@ defmodule Kino.Proxy do
           |> Plug.Conn.send_resp(200, "use /export/data to get extract the report data")
       end)
 
-  Once deployed as an app, the user would be able to export the data
+  Once deployed as an app, the API client would be able to export the data
   by sending a request to `/apps/:slug/proxy/export/data`.
+
+  > #### Authentication {: .warning}
+  >
+  > The paths exposed by `Kino.Proxy` don't use the authentication mechanisms
+  > defined in your Livebook instance.
+  >
+  > If you need to authenticate requests, you should
+  > implement your own authentication mechanism. Here's a simple example.
+  >
+  > ```elixir
+  > Kino.Proxy.listen(fn conn ->
```

Comment:
@josevalim what about this code example with a simple Bearer auth?

```elixir
Kino.Proxy.listen(fn conn ->
  api_token = "my-secret-api-token"

  with ["Bearer " <> client_token] <- Plug.Conn.get_req_header(conn, "authorization"),
       true <- api_token == client_token do
    Plug.Conn.send_resp(conn, 200, "hello")
  else
    _ ->
      conn
      |> Plug.Conn.put_resp_header("www-authenticate", "Bearer")
      |> Plug.Conn.send_resp(401, "Unauthorized")
  end
end)
```

Reply:
```
Kino.Proxy.listen(fn conn ->
  api_token = "my-secret-api-token"

  case Plug.Conn.get_req_header(conn, "authorization") do
    ["Bearer " <> ^api_token] ->
      Plug.Conn.send_resp(conn, 200, "hello")
    _ ->
      conn
      |> Plug.Conn.put_resp_header("www-authenticate", "Bearer")
      |> Plug.Conn.send_resp(401, "Unauthorized")
  end
end)